lib/auth_middleware/setup.py

```diff
@@ -17,8 +17,8 @@
 setup(
     name='auth-middleware',
     version='0.1.0',
-    author='Frank Henard',
-    author_email='fhena2@illinois.edu',
+    maintainer ='Yong Wook Kim',
```

Comment:
```suggestion
    author='Frank Henard',
    maintainer ='Yong Wook Kim',
```

Reply:
sorry for the stupid mistake.... changed.

Reply:
No issues. Thanks.